distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -191,6 +226,28 @@ type Unresolved_Symbol
        Returns the name of an unresolved symbol.
     name self -> Text = @Tail_Call Meta_Helpers.get_unresolved_symbol_name_builtin self.impl
 
+
+    ## ADVANCED
+       GROUP Metadata
+       ICON metadata
+
+       Does this symbol represents an autoscoped constructor?
+
+    is_constructor self -> Boolean = case self of
```

Comment:
If we decide to stick with this `type` representing meta information for all:
- unresolved symbols - e.g. `.name`
- unresolved constructors - e.g. `..Name`
- unresolved conversions

Then we may add even more _"is methods"_, for example:
- `is_method` for symbols that will be resolved to a method invocation
- `is_conversion` for conversions

but we could also got other way...